CHANGELOG.md

```diff
@@ -42,6 +42,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Allows `&Self` as a `#[pymethods]` argument again. [#1071](https://github.com/PyO3/pyo3/pull/1071)
 - Fix best-effort build against PyPy 3.6. #[1092](https://github.com/PyO3/pyo3/pull/1092)
 - Improve lifetime elision in `#[pyproto]`. [#1093](https://github.com/PyO3/pyo3/pull/1093)
+- Fix python configuration detection when cross-compiling. [1077](https://github.com/PyO3/pyo3/issues/1077)
```

Comment:
So far we've always linked against the PR that changes things, not the issue. Though maybe this is ok, @kngwyu?

Reply:
I think PR is better since it also contains the issue link, though I don't have any strong opinion.

Reply:
changed it to the pr for both changes, for the linking against android and for the cross-compilation issue